client/src/language/EmbeddedLanguageDocsManager.ts

```diff
@@ -56,6 +56,21 @@ export default class EmbeddedLanguageDocsManager {
     })
   }
 
+  async deleteEmbeddedLanguageDocsFolder (): Promise<void> {
+    logger.debug(`Delete embedded language documents folder: ${this.embeddedLanguageDocsFolder}.`)
+    await new Promise<void>((resolve) => {
```

Comment:
Use rmSync instead of a complex Promise?

Reply:
We should make sure the LanguageClient's thread never block so it can always answer as fast as possible to the user's actions. The time taken to delete a folder could be perceptible (my external hard drive often gets to sleep). I admit I have absolutely no idea what the user could be doing when this function is called while VS Code is closing, but using the "async" function when making system calls sounds to me like a coding standard we should enforce on the client no matter what.